src/pages/AnalyticsPage/Chart/Chart.tsx

```diff
@@ -0,0 +1,137 @@
+import { format } from 'date-fns'
+import { Area, AreaChart, CartesianGrid, XAxis, YAxis } from 'recharts'
+
+import {
+  type ChartConfig,
+  ChartContainer,
+  ChartLegend,
+  ChartLegendContent,
+  ChartTooltip,
+  ChartTooltipContent,
+} from 'ui/Chart'
+import Icon from 'ui/Icon'
+import LoadingLogo from 'ui/LoadingLogo'
+
+import { useCoverage } from './useCoverage'
+
+// exporting for testing
+export const formatDate = (date: string) =>
```

Comment:
Feel like this could live elsewhere - thoughts? I could definitely seem myself importing this somewhere that's not a Chart